helm-values/tenant-ui/values-sandbox.yaml

```diff
@@ -0,0 +1,56 @@
+ingressSuffix: .apps.silver.devops.gov.bc.ca
+image:
+  tag: "0.2.14"
+  pullPolicy: Always
+traction:
+  pluginInnkeeper:
+    existingSecret: traction-acapy-plugin-innkeeper
+    generated: true
+  apiEndpoint: https://traction-sandbox-tenant-proxy.apps.silver.devops.gov.bc.ca 
+  tenantProxyEndpoint: https://traction-sandbox-tenant-proxy.apps.silver.devops.gov.bc.ca
+  reservation:
+    expiry_minutes: 2880
+    auto_approve:  true
+    auto_issuer: true
+  resources:
+    limits:
+      cpu: 200m
+      memory: 820Mi
+    requests:
+      cpu: 120m
+      memory: 400Mi
+  networkPolicy:
+    enabled: true
+    ingress:
+      enabled: true
+      namespaceSelector:
+        network.openshift.io/policy-group: ingress
+oidc:
+  active: false
+  showInnkeeperAdminLogin: false
+  showWritableComponents: false
+  authority: https://dev.loginproxy.gov.bc.ca/auth/realms/digitaltrust-nrm
+  jwksUri: https://dev.loginproxy.gov.bc.ca/auth/realms/digitaltrust-nrm/protocol/openid-connect/certs
+ariesDetails:
+  ledgerDescription: "bcovrin-test"
+smtp:
+  server: apps.smtp.gov.bc.ca
+  port: 25
+  senderAddress: DoNotReplyTractionSandbox@gov.bc.ca
+  innkeeperInbox: "lucas.o'neil@gov.bc.ca,emiliano.sune@quartech.com"
```

Comment:
I thought this comment was just referring to innkeeperInbox, but is it intended for the whole block here?

I think ledgerDescription should stay, I think keep the top 2 OIDC fields as well maybe?

Reply:
I didn;t see `ledgerDescription` 😅, I was focused on the `oidc` and `smtp` blocks. Are `showInnkeeperAdminLogin` and `showWritableComponents` required if oidc is not enabled?

Reply:
`showWritableComponents` and `showInnkeeperAdminLogin` should each be true yeah

I guess the OIDC parts (`authority` and `jwksUri`) could be blanked, just that I'm not sure I ever tried with them as blank strings I think... might as well try it out here

Reply:
@esune @loneil

Reply:
```
ingressSuffix: .apps.silver.devops.gov.bc.ca
image:
  tag: "0.2.14"
  pullPolicy: Always
traction:
  pluginInnkeeper:
    existingSecret: traction-acapy-plugin-innkeeper
    generated: true
  apiEndpoint: https://traction-sandbox-tenant-proxy.apps.silver.devops.gov.bc.ca 
  tenantProxyEndpoint: https://traction-sandbox-tenant-proxy.apps.silver.devops.gov.bc.ca
  reservation:
    expiry_minutes: 2880
    auto_approve:  true
    auto_issuer: true
  resources:
    limits:
      cpu: 200m
      memory: 820Mi
    requests:
      cpu: 120m
      memory: 400Mi
  autoscaling:
    enabled: false
    minReplicas: 1
    maxReplicas: 10
    targetCPUUtilizationPercentage: 80
    targetMemoryUtilizationPercentage: 80
  networkPolicy:
    enabled: true
    ingress:
      enabled: true
      namespaceSelector:
        network.openshift.io/policy-group: ingress
oidc:
  active: false
  showInnkeeperAdminLogin: false
  showWritableComponents: false
  authority: ""
  jwksUri: ""
ariesDetails:
  ledgerDescription: "bcovrin-test"
smtp:
  server: apps.smtp.gov.bc.ca
  port: 25
  senderAddress: DoNotReplyTractionSandbox@gov.bc.ca
  innkeeperInbox: ""
resources:
  limits:
    cpu: 400m
    memory: 1600Mi
  requests:
    cpu: 200m
    memory: 820Mi
ingress:
  annotations:
    route.openshift.io/termination: edge
networkPolicy:
  enabled: true
  ingress:
    enabled: true
    namespaceSelector:
      network.openshift.io/policy-group: ingress
      ```

Reply:
> `showWritableComponents` and `showInnkeeperAdminLogin` should each be true yeah
> 
> I guess the OIDC parts (`authority` and `jwksUri`) could be blanked, just that I'm not sure I ever tried with them as blank strings I think... might as well try it out here

I thought `showWritableComponents` and `showInnkeeperAdminLogin` would be set to `false` since the innkeeper is really not being used?

Reply:
@rajpalc7 please don't post file changes as a comment, once a decision has been made and changes are needed just add a commit to the PR so we can see the diff in the "files changed" tab

Reply:
Sounds good emiliano, I have created new commit for now so you can see the diff easily. https://github.com/bcgov/trust-over-ip-configurations/pull/146/commits/10fb3891ca1a72535fbad6dd76d86904dd744ea2

Reply:
`showWritableComponents` will have to be true, it's just a flag to remove buttons and things for issuance. So not really tied to Innkeeper, even if the auto issuer stuff is on. Unless people aren't going to be allowed to create creds or issue or stuff in the Sandbox.

`showInnkeeperAdminLogin` would be true if we want people to be able to log into the Innkeeper without the IDIR login (which would be necessary as the OIDC connections aren't being configed here), assuming we still want someone operating the sandbox to be able to go in as the Innkeeper to see Tenants and system things, even if they don't need to approve.

Reply:
Gotcha, thanks for the clarification. @rajpalc7 see comments above from Lucas.